packages/shopify-codemod/transforms/utils.js

```diff
@@ -14,6 +14,24 @@ export function findLastMember(node) {
   return node;
 }
 
+export function forEachAssignment(assignment, cb) {
+  if (assignment.right && assignment.left) {
+    if (j.AssignmentExpression.check(assignment.right)) {
+      let rightName;
+      if (assignment.left.type === 'Identifier') {
+        rightName = j.identifier(assignment.right.left.name);
+      } else {
+        rightName = j.memberExpression(assignment.right.left.object, assignment.right.left.property);
+      }
+      forEachAssignment(assignment.right, cb);
+      return cb(j.assignmentExpression('=', assignment.left, rightName));
```

Comment:
I would not expect that this helper is creating additional assignment expressions for me.